src/dynamodb.py

```diff
@@ -6,17 +6,29 @@
 import os
 import argparse
 
-
 def delete_all_items(table):
+    """
```

Comment:
Use Google-style to comment functions. More information [here](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)